cinema/views.py

```diff
@@ -167,6 +180,26 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="date",
+                type=OpenApiTypes.DATE,
+                description="Filter movie sessions by date "
+                            "(ex. ?date=2024-10-09)",
+            ),
+            OpenApiParameter(
+                name="movie",
+                type=OpenApiTypes.INT,
+                description="Filter movie sessions by movie id "
+                            "(ex. ?movie=4,5)",
```

Comment:
The description for the `movie` filter suggests multiple IDs can be passed (e.g., ?movie=4,5), but the current implementation only handles a single ID. Consider updating the description or modifying the code to handle multiple IDs.